microsoftgraph/microsoft-graph-toolkit
# makes mgt-person screen-readable

Closes # 

### PR Type


Bugfix







### Description of the changes
Makes the person component screen readable.

### PR checklist

### Other information


Fixes #1104

### How to test
* Pull these changes and follow steps in the video below
* Observe that Narrator mentions the peron's name when their image is focused on

https://user-images.githubusercontent.com/33054846/117290074-00ada800-ae8b-11eb-9b9d-31d8145a2a1f.mp4
